metrics/linux-gcc8-release/render-tests/sprites/array-default-only/metrics.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file should be also in `ios-render-test-runner` in order to be found on iOS platform.

Reply:
And in `android-render-test-runner` for Android platform.